src/lib/markbind/src/parsers/componentParser.js

```diff
@@ -119,6 +121,41 @@ function _assignPanelId(element) {
   }
 }
 
+/**
+ * Check and warns if element has inconsistent attributes.
+ * @param element Root element to check
+ * @param attribute An attribute that is inconsistent with other attributes
+ * @param attrsInconsistentWith The attributes inconsitent with `attribute`
+ */
+function _warnInconsistentAttributes(element, attribute, attrsInconsistentWith) {
+  if (attribute in element.attribs) {
+    for (let i = 0; i < attrsInconsistentWith.length; i += 1) {
+      if (attrsInconsistentWith[i] in element.attribs) {
+        // TODO: Use logger here instead of console.log. See issue #1060
+        // eslint-disable-next-line no-console
+        console.log(`warn: Usage of inconsistent ${element.name} attributes: `
+          + `'${attribute}' with '${attrsInconsistentWith[i]}'`);
+        return;
+      }
+    }
+  }
+}
```

Comment:
I wrote the function like so because i noticed that some attributes are inconsistent with a set of attributes. 
Like A-B, A-C, A-D
At first I wanted to check for the pairs individually, but I felt that cutting short the search at the absence of A would be good.